src/components/Auth.vue

```diff
@@ -1,38 +1,44 @@
 <template>
-  <v-layout row align-baseline>
-    <v-btn v-if = "!loggedIn" outline round color = "primary" @click="loginUser">
+  <v-layout row align-center>
+    <v-btn v-if="!loggedIn" outline round color="primary" @click="loginUser">
       Login
     </v-btn>
-    <v-btn v-if = "loggedIn" outline round color = "primary" @click = "logoutUser">
+    <v-btn v-if="loggedIn" outline round color="primary" @click="logoutUser">
       Logout
     </v-btn>
-    <v-tooltip bottom :disabled = "saveWarnings.length===0">
-      <v-icon slot = "activator" v-if = "!currentlySaving && !gettingUserData" :color = "saveColor">
+    <v-tooltip bottom :disabled="saveWarnings.length===0">
+      <v-icon
+        slot="activator"
+        v-if="!currentlySaving && !gettingUserData"
+        :color="saveColor"
+        style="user-select: none;"
+      >
         {{saveIcon}}
       </v-icon>
       <div>
-        <p v-for = "saveWarning in saveWarnings">{{saveWarning.name}}: {{saveWarning.error}}</p>
+        <p v-for="saveWarning in saveWarnings">{{saveWarning.name}}: {{saveWarning.error}}</p>
       </div>
     </v-tooltip>
-    <div v-if = "currentlySaving || gettingUserData">
-      <v-progress-circular :size = "18" indeterminate>
+    <div v-if="currentlySaving || gettingUserData">
+      <v-progress-circular :size="18" indeterminate>
       </v-progress-circular>
     </div>
   </v-layout>
 </template>
 
 <script>
-import Vue from 'vue'
-import UAParser from "ua-parser-js"
-var DATE_FORMAT = "YYYY-MM-DDTHH:mm:ss.SSS000Z"
+import Vue from 'vue';
+import UAParser from "ua-parser-js";
+var DATE_FORMAT="YYYY-MM-DDTHH:mm:ss.SSS000Z";
+var FIREROAD_LINK="https://fireroad-dev.mit.edu";
```

Comment:
I think this line got caught in a merge from somewhere else, it's no longer needed.